### DISCUSSION TITLE
Using my own function in code

### DISCUSSION THREAD

**Discussion by A-ep93** [anyoptimization/pymoo#223] [Q&A]
Hello all,
I am new to pymoo. I want to use the NSGA-II tutorial which is posted [here](https://pymoo.org/algorithms/moo/nsga2.html#nb-nsga2). However, I am not sure how can I implement my own custom function in it? How should I define that function? Any examples to share?

**Comment by drpeterallan** [comment]
I found this youtube [tutorial](https://www.youtube.com/watch?v=six6afYc7RI) very useful to get me started.

**Comment by ubeydtalha** [comment]
I think it is very well explained on the website of the library.
https://pymoo.org/getting_started/part_1.html
